src/background/certificate/providers/ServerProvider.ts

```diff
@@ -6,10 +6,23 @@ import { CodedError } from "../../../shared/types/errors/CodedError";
 import { ErrorFactory } from "../factories/ErrorFactory";
 import { CertificateProvider } from "./CertificateProvider";
 
-const SERVER_URL =
-  "https://snonitze65.execute-api.eu-central-1.amazonaws.com/getCertificate/";
-
 export class ServerProvider implements CertificateProvider {
+  private defaultServerUrl =
+    "https://snonitze65.execute-api.eu-central-1.amazonaws.com/getCertificate/";
```

Comment:
http://localhost:3000/getCertificate/
-> The user does not want to know the implementation good enough to be able to determine the endpoint. It would be better to split the endpoint from the url.

Reply:
This is also better for the future since there may be more endpoints then.

Reply:
Implemented.